executor/executor_test.go

```diff
@@ -178,6 +178,13 @@ func (s *testSuite) TestAdmin(c *C) {
 	c.Assert(row.GetString(1), Equals, historyJobs[0].State.String())
 	c.Assert(err, IsNil)
 
+	// show DDL job queries test
+	r, err = tk.Exec("admin show ddl job queries 1")
+	c.Assert(err, IsNil)
+	row, err = r.Next(ctx)
+	c.Assert(err, IsNil)
+	c.Assert(row.Len(), Equals, 1)
```

Comment:
I think a better way is:
```go
r = tk.MustQuery("admin show ddl job queries 1")
r.Check(testkit.Rows(...))
```
And please add some more complicated test cases, like:
```go
admin show ddl job queries 1, 1, 1;
admin show ddl job queries 1, 2, 3, 4;
admin show ddl job queries 1.12, 1.121, 0.1;
admin show ddl job queries -11, -12, 0, 100;
```

Reply:
Agree I added

Reply:
@ciscoxll You see the other test cases in TestAdmin.